src/components/stack/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+const Stack = ({
+	additionalClassNames,
+	alignment,
+	children,
+	direction,
+	divider,
+	flexType,
+	justification,
+	gap,
+	wrap,
+}) => {
+	const defaultAndAdditionalClassnames = `c-stack${additionalClassNames ? ` ${additionalClassNames}` : ''}`;
+	return (
+		<div
+			className={defaultAndAdditionalClassnames}
+			data-style-direction={direction}
+			data-style-justification={justification}
+			data-style-alignment={alignment}
+			data-style-flextype={flexType}
+			data-style-wrap={wrap}
+			style={{ '--c-stack-gap': gap }}
+		>
+			{ Array.isArray(children)
+				? children.map((child) => (
```

Comment:
When dealing with children in React and needing to iterate on them React provides methods and functions that should be used - https://reactjs.org/docs/react-api.html#reactchildren

Using `React.Children` also removes the need and issues with keys

```jsx
			{React.Children.map(children, (child) => (
				<>
					{child}
					{divider ? <hr aria-hidden="true" /> : null}
				</>
			))}
```

Reply:
Wow!  Awesome!